### PR TITLE
Database

### PR DESCRIPTION
**Pull request by ADAdebomi**
Added database functionality, please review and test for any edge cases you could think of

### CODE REVIEW

File: src/Database/AirportDatabase.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package Database;
+
+import Model.Airport;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class AirportDatabase {
+    HashMap<String,Airport> airpotsDb;
```
Comment (emn3779) [inline src/Database/AirportDatabase.java]: airports spelling

**Comment by emn3779** [comment]
Looks good, still a few todo's but one functionality at a time
